tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -5,20 +5,37 @@
 Repairing artifacts with regression
 ===================================
 
-This tutorial covers removal of artifacts using regression as in
-:footcite:`GrattonEtAl1983`.
+This tutorial covers removal of artifacts using regression as in Gratton et al.
+1983 :footcite:`GrattonEtAl1983`.
 
 .. contents:: Page contents
    :local:
    :depth: 2
 
+Generally speaking, artifacts that result in time waveforms on the sensors
+that are accurately reflected by some reference signal can be removed by
+regression. One example of this is blink artifacts captured by (typically
```

Comment:
"is blink artifacts" sounds a bit strange - "are blink artifacts" is maybe better ("Blink artifacts captured by...are one example of this").